### PR TITLE
added database

### PR DESCRIPTION
**Pull request by mgirar84**

### CODE REVIEW

File: collection_2020-03-24.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Dump of table roadbikes
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `roadbikes`;
+
+CREATE TABLE `roadbikes` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `model` varchar(11) DEFAULT NULL,
+  `brand` varchar(11) DEFAULT NULL,
+  `price` int(11) DEFAULT NULL,
```
Comment (ashleycoles) [inline collection_2020-03-24.sql:32]: Could the price potentially be a float? What if it costs 1000.99 or something like that?

File: collection_2020-03-24.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+# Dump of table roadbikes
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `roadbikes`;
+
+CREATE TABLE `roadbikes` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `model` varchar(11) DEFAULT NULL,
```
Comment (ashleycoles) [inline collection_2020-03-24.sql]: Well done for setting field lengths here, but will 11 characters be enough for the model/ brand?

**Review by ashleycoles — COMMENTED** [review]
Only the one comment for you! Well done for setting the lengths of your fields

File: collection_2020-03-24.sql (hunk 1 above)
Comment (mgirar84) [reply collection_2020-03-24.sql:32]: good point, though I don't think I've seen any prices listed with decimals.

File: collection_2020-03-24.sql (hunk 2 above)
Comment (mgirar84) [reply collection_2020-03-24.sql]: I think it's safe for brand, but model could be longer. Honestly the field lengths are just the defaults.

**Pull request merged**